mobify/selector-utils
# Bring tool up-to-date

Status: **Ready for Review**
Owner: @marlowpayne
Reviewers: @fractaltheory @jansepar @mikenikles 

## Changes
This repo is in a serious state of neglect. Let's help bring things more up-to-date with current standards:
- Bring dependencies up-to-date with current Adaptive.js standards
- Commit required bower components
- Update linting to current standards
- Fix lints
- Add jQuery in anticipation of fixing #16 
- Add a `CHANGELOG` for gitflow / releases
- Update `README`

## Todos:

### Feedback:
_none so far_

## How to Test
- Checkout code
- `grunt test`
- Link to this branch in an Adaptive.js project that uses selector-utils + Zepto, make sure selector utils still function correctly in project
